scripts/missions/sandoria/6_2_Ranperres_Final_Rest.lua

```diff
@@ -188,6 +190,8 @@ mission.sections =
                         not player:hasKeyItem(xi.ki.ANCIENT_SAN_DORIAN_BOOK)
                     then
                         return mission:progressEvent(8)
+                    elseif player:hasKeyItem(xi.ki.ANCIENT_SAN_DORIAN_BOOK) then
+                        return mission:messageSpecial(krtID.text.FINAL_RESTING_PLACE)
```

Comment:
`krtID.text.FINAL_RESTING_PLACE` needs an entry in IDs.

Reply:
great catch -